packages/exo/src/exo-tools.js

```diff
@@ -40,6 +47,15 @@ const defendSyncArgs = (args, methodGuardPayload, label) => {
     restArgGuard,
   );
   mustMatch(harden(args), paramsPattern, label);
+  if (restArgGuard !== undefined) {
+    return args;
+  }
+  const declaredLen =
+    argGuards.length + (optionalArgGuards ? optionalArgGuards.length : 0);
+  if (args.length <= declaredLen) {
+    return args;
+  }
+  return args.slice(0, declaredLen);
```

Comment:
consider documenting,
```suggestion
  // ignore extraneous arguments, as a JS function call would do
  return args.slice(0, declaredLen);
```

Reply:
Done.